### PR TITLE
Fixes #28339 - update eslint

### PR DESCRIPTION
**Pull request by amirfefer**
Part of the `foreman-env`

### CODE REVIEW

**Comment by sharvit** [comment]
Thanks @amirfefer, notice the Travis error about `jest`

**Comment by amirfefer** [comment]
@sharvit any idea why it breaks on node 12?

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
   - '10'
-  - '12'
+  - '12.13.0'
```
Comment (amirfefer) [inline .travis.yml]: the `/.bin/jest` module loading error  is only on node 12.13.1...

Comment (sharvit) [reply .travis.yml]: Nice catch :+1: 

Should we add a comment and/or a redmine issue to track it? Assuming we would want to bring it back to 12 once they fix the issue.

Comment (amirfefer) [reply .travis.yml]: I added a comment for it

**Comment by amirfefer** [comment]
@ekohl / @evgeni  could we get an ack, please?

**Review by evgeni — APPROVED** [review]
with my @theforeman/packaging powers: ACK, we don't care/package eslint

**Comment by mmoll** [comment]
builds still fail?

**Comment by sharvit** [comment]
> builds still fail?

Tests failures are unrelated, Travis actually never ran `eslint` because of the testing errors.


BTW, why can't I tag @theforeman/packaging in github as you can? :confused:  (@evgeni)

**Comment by mmoll** [comment]
Could develop get fixed first, please? :)

**Comment by sharvit** [comment]
> Could develop get fixed first, please? :)

It is tricky atm since the issue comes from `react-bootstrap`, afaik @laviro is trying to fix that.
BTW, we don't have this issue in production (or in a development **server**) since prebuild the vendor and we use `lock` files.

IMO we can merge this PR if `eslint` works locally.

File: webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-unused-vars */
+// eslint bug - https://github.com/eslint/eslint/issues/12117
```
Comment (sharvit) [inline webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js:2]: Some developers suggested using `eslint v6.7.1` and some suggest to update `babel-eslint` to `10.0.3`

Comment (amirfefer) [reply webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js:2]: doesn't work :/
```
/home/afeferku/git/foreman/webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js
  13:14  error  'item' is defined but never used   no-unused-vars
  14:16  error  'child' is defined but never used  no-unused-vars
```

**Review by sharvit — APPROVED** [review]
Thanks @amirfefer

**Pull request merged**